### DISCUSSION TITLE
Why doesn't have IMQImpl in the master?

### DISCUSSION THREAD

**Discussion by BugCui** [rabbitmq/rabbitmq-java-client#685] [General]
Why doesn't have IMQImpl in the master?

**Comment by michaelklishin** [comment]
I will convert this issue to a GitHub discussion. Currently GitHub will [automatically close and lock the issue](https://github.com/github/feedback/discussions/3197) even though your question will be transferred and responded to elsewhere. This is to let you know that we do not intend to ignore this but this is how the current GitHub conversion mechanism makes it seem for the users :(

**Comment by michaelklishin** [comment]
I don't understand the question. Consider elaborating. The entire history of this project is public and available in this repository.

**Comment by acogoluegnes** [comment]
Some code is generated and is not committed into the repository, maybe this is what you're referring to.

**Reply by BugCui** [reply]
I understand，thank you！
